staff/AbelusM/css-layout-simple-adaptive/styles/desktop.css

```diff
@@ -0,0 +1,50 @@
+
+* {
```

Comment:
TIP move common style rules (mobile & desktop) to main.css

Reply:
done!